docs/sample_collection.rst

```diff
@@ -0,0 +1,63 @@
+****************
+SampleCollection
+****************
+
+Data export, analysis and visualization functions are all contained within the
+``SampleCollection`` class.
+
+A ``SampleCollection`` is returned whenever multiple Samples returned via the
```

Comment:
*A SampleCollection is returned whenever multiple Samples are returned via the One Codex API using a model.*